x-pack/filebeat/input/cel/cel_metrics_test.go

```diff
@@ -0,0 +1,168 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+package cel
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/elastic/beats/v7/x-pack/filebeat/otel"
+	"github.com/elastic/elastic-agent-libs/logp"
+	"github.com/elastic/elastic-agent-libs/monitoring"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Please don't use testify in the cel package for the reasons outlined in [this](https://go.dev/wiki/TestComments#assert-libraries).